wasm/src/language_service.rs

```diff
@@ -178,6 +208,47 @@ serializable_type! {
     IDefinition
 }
 
+serializable_type! {
+    SignatureHelp,
+    {
+        signatures: Vec<SignatureInformation>,
+        active_signature: u32,
+        active_parameter: u32,
```

Comment:
We should be able to use camelCase now without clippy yelling at us. (Another style consistency thing for this file)

Reply:
Thank goodness!